src/ports/catalog/queries.ts

```diff
@@ -582,6 +582,10 @@ const getTradesCTE = () => {
           LEFT JOIN ${MARKETPLACE_SQUID_SCHEMA}.nft AS nft ON (ta.contract_address = nft.contract_address AND erc721_asset.token_id::numeric = nft.token_id)
           LEFT JOIN ${MARKETPLACE_SQUID_SCHEMA}.account as account ON (account.id = nft.owner_id)
         ) AS assets_with_values ON t.id = assets_with_values.trade_id
+         AND (
+          assets_with_values.direction != 'sent'
```

Comment:
This change seems to be filtering by `asset` and a trade has multiple assets which are later grouped to create a trade. If we filter the assets, trades without the assets that are not owned by the owner of the trade will be created as a result of the query. I might be missing something, but that's why we used the `HAVING` at the end of the query.

Reply:
Oh, you're right, I forgot to push the change from below, that adds:
```
AND assets_with_values.direction = 'sent'
```
in the main WHERE, filtering out the trades with direction NOT 'sent'. The 'sent' with NO values won't make it in the query since they are filtered out in the JOIN. 

I tried this out with the new trades subquery and the old one with the HAVING and they both return 3.761 trades atm, so we're getting the same amount. (Without AND assets_with_values.direction = 'sent' it indeed returns more, like 4200 results)